package.json

```diff
@@ -144,7 +144,6 @@
     },
     "overrides": {
       "ember-auto-import": "^2.10.0",
-      "@embroider/macros": "^1.16.6",
```

Comment:
without this change the wrong version of macros ends up being used (along with the wrong shared-internals, etc)